docs/ops/detection/ROIPooling_1.md

```diff
@@ -44,13 +50,22 @@
 
 **Inputs**:
 
-*   **1**: 4D input tensor of shape `[1, C, H, W]` with feature maps. Required.
+*   **1**: 4D input tensor of shape `[N, C, H, W]` with feature maps of type *T*. Required.
+
+*   **2**: 2D input tensor of shape `[NUM_ROIS, 5]` describing region of interest box consisting of 5 element tuples of type *T*: `[batch_id, x_1, y_1, x_2, y_2]`. Required.
```

Comment:
These changes are rendered very poorly. Let's move detailed description of the input format to the `detailed description` section